requirements/base.in

```diff
@@ -14,6 +14,6 @@ django-choices==1.7.0
 django-cors-middleware
 django-markup==1.3
 django_redis
-
+factory-boy==3.0.1
```

Comment:
Deze dependecy is niet nodig voor alle omgevingen, wil je waarschijnlijk aan `testing.in` toevoegen